integreat_cms/deepl_api/utils.py

```diff
@@ -42,6 +43,38 @@ def check_availability(request, language_slug):
             and language_slug in deepl_config.supported_target_languages
         )
 
+    def check_usage(self, region, source_translation):
+        """
+        This function checks if the attempted translation would exceed the region's word limit
+
+        :param region: region for which to check usage
+        :type region: ~integreat_cms.cms.models.regions.region.Region
+
+        :param source_translation: single content object
+        :type source_translation: ~integreat_cms.cms.models.abstract_content_translation.AbstractContentTranslation
+
+        :return: translation would exceed limit, attempted translation word count
+        :rtype: (bool, int)
+        """
+        # Gather content to be translated and calculate total word count
+        content_to_translate = []
+        for attr in self.translatable_attributes:
+            if hasattr(source_translation, attr):
+                content_to_translate.append(getattr(source_translation, attr))
+        content_to_translate = " ".join(content_to_translate)
+        for char in "-,.\n":
+            content_to_translate = content_to_translate.replace(char, " ")
+        word_count = len(content_to_translate.split())
```

Comment:
As mentioned in the issue description, I guess this will be almost certainly wrong. A better estimation would maybe be to use [`strip_tags()`](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.html.strip_tags), but then again I'm not sure if we're using much more of DeepLs credits than what we charge our own customers.
Would also be interesting to know how tinymce calculates the words in the editor^^

Reply:
I unfortunately wasn't able to find the source code for the TinyMCE plugin responsible, but I think your `strip_tags()` suggestion plus the `tang_handling="html"` option should probably be accurate enough

Reply:
Sounds good! :+1: